ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -193,6 +193,19 @@ else
   echo "zones already set in install-config.yaml, skipped"
 fi
 
+if [[ -n "${AWS_EDGE_POOL_ENABLED-}" ]]; then
+  mapfile -t LOCAL_ZONES < <(aws --region "${aws_source_region}" ec2 describe-availability-zones \
+    --filter Name=state,Values=available Name=zone-type,Values=local-zone \
+    | jq -r '.AvailabilityZones[].ZoneName' | sort -u | tail -n 1)
+  LOCAL_ZONES_STR="[ $(join_by , "${LOCAL_ZONES[@]}") ]"
+  export EDGE_COMPUTE_POOL_CONFIG="
```

Comment:
This probably won't work on jobs where AWS_EDGE_POOL_ENABLED is disabled, as `EDGE_COMPUTE_POOL_CONFIG` will crash due to `set -o nounset`.

Lets create another yq patch if edge pool is set?

Reply:
Yep, that's a good idea to create the yq patch, thanks! I was unsure if it will merge a list (compute pool).

Not sure why didn't crash in the [rehearsal `e2e-aws-ovn`](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40031/rehearse-40031-pull-ci-openshift-installer-master-e2e-aws-ovn/1665838316009820160), maybe due `${EDGE_COMPUTE_POOL_CONFIG-}`.